docs/Arguments.md

```diff
@@ -104,15 +107,15 @@ usage: -m llumnix.entrypoints.vllm.api_server [-h]
 `--scaling-policy`
 - Scaling policy.
 - Possible choices: max_load, avg_load
-- default: "max_load"
+- default: "avg_load"
 
 `--scale-up-threshold`
 - Scale up threshold.
-- Default: 4
+- Default: 10
 
 `--scale-down-threshold`
 - Scale down threshold.
-- Default: 100
+- Default: 60
```

Comment:
why change these default values?

Reply:
The default values in Arguments.md and arg_utils.py is not consistent. So I change the default values in Arguments.md to be consistent with the values in arg_utils.py.

Reply:
I see